code/game/gamemodes/clock_cult/clock_scriptures/scripture_applications.dm

```diff
@@ -89,6 +89,30 @@
 	quickbind_desc = "Creates a Sigil of Transmission, which can drain and will store power for clockwork structures."
 
 
+//Clockwork Marauder: Creates a construct shell for a clockwork marauder, a well-rounded frontline fighter.
+/datum/clockwork_scripture/create_object/construct/clockwork_marauder
+	descname = "Well-Rounded Combat Construct"
+	name = "Clockwork Marauder"
+	desc = "Creates a shell for a clockwork marauder, a balanced frontline construct."
+	invocations = list("Arise, avatar of Neovgre!", "Defend this place and this Ark.")
+	channel_time = 50
+	consumed_components = list(BELLIGERENT_EYE = 4, GEIS_CAPACITOR = 2, REPLICANT_ALLOY = 2)
+	creator_message = "<span class='brass'>Your slab disgorges several chunks of replicant alloy with form into a suit of thrumming armor.</span>"
+	usage_tip = "The marauder's shield can effectively deflect energy-based projectiles."
+	tier = SCRIPTURE_APPLICATION
+	one_per_tile = TRUE
+	primary_component = REPLICANT_ALLOY
```

Comment:
Given that it created an empty construct shell, I wasn't entirely certain which one I wanted to go with.